diffusers/Prompt Saving Callback.ipynb

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
<p>Replace with:</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># Prompt scheduling callback

Prompt scheduling callback allows changing prompts during a generation, like&nbsp;[prompt editing in A1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#prompt-editing).



</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/525/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>